frontend/resourceadm/components/ResourceDeployStatus/ResourceDeployStatus.tsx

```diff
@@ -74,7 +74,7 @@ export const ResourceDeployStatus = ({
             <LinkButton onClick={() => onNavigateToPageWithError(e.pageWithError)}>
               {textArr[1]}
             </LinkButton>
-            {'"'}
+            {'"' + textArr[2]}
```

Comment:
Why do we need the '"'"? 🤔

Reply:
To make it look like this: 
<img width="271" alt="Screenshot 2023-08-29 at 08 03 06" src="https://github.com/Altinn/altinn-studio/assets/133344438/019a7386-fe41-43d4-9ed6-17988513b706">